components/card/card.js

```diff
@@ -1,29 +1,23 @@
-export function createCharacterCard({
```

Comment:
I moved back to passing the entire character, leveraging the response from the api. If in the future we want to add other data, it is already available. The code is also more readable both here and in the index.